### DISCUSSION TITLE
Can't open another file while cursor is on fugitive buffer

### DISCUSSION THREAD

**Discussion by amirHossein5** [tpope/vim-fugitive#2275] [General]
When my cursor is on fugitive buffer, and I try to open another file(via `:e file.txt`), I get:

`E1513: Cannot switch buffer. 'winfixbuf' is enabled`

**Comment by tpope** [comment]
This was a deliberate change. See https://github.com/tpope/vim-fugitive/issues/2272. Rolling back isn't out of the question.

**Reply by leafgarland** [reply]
It might just be my workflow but using winfixbuf for the general/main fugitive buffer makes things awkward as I often switch to fugitive (`:0Git`) and then back to my previous buffer, which is now blocked. I can quit the fugitive window (`gq`) but that closes the window, and it was my window not fugitive's :)

**Reply by leafgarland** [reply]
In a single window layout, quitting fugitive (`gq`) closes the buffer but leaves the winfixbuf option set. So then later I hit the same problems, even though I'm not viewing a fugitive buffer.

The winfixbuf option seems like something that should only be used when creating a window (like the blame view).

**Reply by tpope** [reply]
The preferred way to open the status buffer directly in the current window is `:Gsplit :` (or `:Gsp:` for short). If you do that, it won't set `'winfixbuf'`.

`:0Git` is an accident. I added `0` support for commands like `:0Git log`. I hadn't considered the status buffer when I did that.

**Reply by leafgarland** [reply]
Hm, `:Gsplit :` splits the window for me, but I guess that is something in my config and I can investigate. Thanks.

**Reply by tpope** [reply]
I meant to say `:Gedit :`.

**Reply by leafgarland** [reply]
That works perfectly, thanks for the guidance.

**Reply by amirHossein5** [reply]
Also, it's not possible to open files using fuzzy finders

**Reply by tpope** [reply]
> Also, it's not possible to open files using fuzzy finders

That's the intended behavior. We're saying, "This window is for the Fugitive status only. Get lost, fuzzy finders."
